packages/cli/src/runDeploy.ts

```diff
@@ -19,6 +19,10 @@ import { configToModules } from "./deploy/configToModules";
 import { findContractArtifacts } from "@latticexyz/world/node";
 import { enableAutomine } from "./utils/enableAutomine";
 
+function encodeToHex(str: string): Hex {
+  return ("0x" + Buffer.from(str, "utf8").toString("hex")) as Hex;
+}
+
```

Comment:
there is a util from `viem` we can use instead: https://viem.sh/docs/utilities/toHex.html